bin/ncp/CONFIG/nc-nextcloud.sh

```diff
@@ -203,6 +203,26 @@ EOF
 </VirtualHost>
 EOF
 
+  # for notify_push app in NC21
+  a2enmod proxy
+  a2enmod proxy_http
+  a2enmod proxy_wstunnel
+
+  cat > /etc/systemd/system/notify_push.service <<EOF
+[Unit]
+Description = Push daemon for Nextcloud clients
+
+[Service]
+Environment=PORT=7867
+Environment=NEXTCLOUD_URL=https://localhost
```

Comment:
I hardcoded the url, because there's no harm in using localhost and otherwise, we would have to restart the systemd service every time we changed the NC url

Reply:
are we sure that this doesn't make it only accessible on localhost? I think that `ncc notify_push:setup` complaints about that

Reply:
etc)

Reply:
That value is only how notify_push connects to Nextcloud (probably for retrieving the list of notifications, etc.)
This configuration is actually right from an example in the notify_push repo

Reply:
Just looked it up again: It's [in the Readme](https://github.com/nextcloud/notify_push#troubleshooting) under Troubleshooting:

> If your nextcloud hostname resolves do a dynamic ip you can try setting the NEXTCLOUD_URL to the internal ip of the server.

That's why I think it's both unproblematic and beneficial for us to use localhost (notify_push might not handle dynamic IPs very well)